src/views/subscribe/subscribe.js

```diff
@@ -0,0 +1,235 @@
+
+  const FormFields = {
+    firstName: '',
+    lastName: '',
+    email: '',
+  };
+
+  const Success = () => {
+    return (
+      <div
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Styled.h4 sx={{ color: 'white' }}>
+          You have successfully subscribed to OpenEBS Newsletter.
+        </Styled.h4>
+      </div>
+    )
+  }
+
+  const Failed = () => {
+    return (
+      <div
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Styled.h4 sx={{ color: 'white' }}>
+          Error in submitting the form, please try again!!!
+        </Styled.h4>
+      </div>
+    )
+  }
+
+  const sendData = (values, actions) => {
+    // TODO: Set up modular popup
+    if (values.email === '' || values.email === 'undefined') {
+      console.log(`please enter valid email`)
+    } else {
+      async function postData(url = '', data = {}) {
+        const response = await fetch(url, {
+          method: 'POST',
+          mode: 'cors',
+          cache: 'no-cache',
+          credentials: 'same-origin',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          redirect: 'follow',
+          referrerPolicy: 'no-referrer',
+          body: JSON.stringify(data),
+        })
+        return await response.json()
+      }
+      postData(GATSBY_NEWSLETTER_BACKEND_URL, {
+        firstname: values.firstName,
+        lastname: values.lastName,
+        email: values.email,
+        tag: tag,
+      })
+        .then((data) => {
+          setIsSuccess(true)
+          handleOpenModal()
+          actions.resetForm({
+            values: FormFields
+          });
+        })
+        .catch((err) => {
+          console.error(err)
+          setIsSuccess(false)
+          handleOpenModal()
+        })
+    }
+  }
+
+  const handleOpenModal = () => {
+    setModal(true)
+  }
+
+  const handleCloseModal = () => {
+    setModal(false)
+  }
+
+  const formik = useFormik({
+    initialValues: FormFields,
+    validate: values => {
+      const errors = {};
+
+      if (!values.email) {
+        errors.email = 'Please complete this required field.';
+      } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
+        errors.email = 'Invalid email address.';
+      }
+
+      return errors;
+    },
+    onSubmit: (values, actions) => {
+      sendData(values, actions)
+    },
+  });
+  
+  const { handleChange, values, handleBlur, handleSubmit, touched, errors } = formik;
+  
+  return (
+    <>
+      <Container sx={{ p: 4 }}>
+        <div className="row subscribe-class">
+          <div className="col-md-6 col-sm-12">
+            <div className="heading-class">
+              <span>
+                Get all the OpenEBS updates & information straight to your inbox 📨
```

Comment:
I will suggest to use svg or icon from library.

Reason:

That icon which are provided in the text will vary based on browser.